dataline-integrations/singer/exchangerateapi_io/source/entrypoint.py

```diff
@@ -0,0 +1,23 @@
+import os
+import sys
+
+from dataline_entrypoint import DatalineEntrypointBuilder
+
+def spec_handler(args):
+    print("spec") # todo
```

Comment:
I didn't actually implement the spec retrieval yet, that seems like it will require quite a bit more work.